packages/cli/src/commands/base.ts

```diff
@@ -14,7 +15,7 @@ export abstract class BaseCommand extends Command<Context> {
     this.context.configu = configu;
 
     // register builtins here
-    Registry.register({ NoopConfigStore, InMemoryConfigStore });
+    Registry.register({ NoopConfigStore, InMemoryConfigStore, XmlFileConfigStore });
```

Comment:
the xml might not be one of the built-in integrations.
please remove from the CLI.